server/src/typeDefs/mutation.ts

```diff
@@ -8,5 +8,8 @@ export const mutation = gql`
     forgetPassword(email: String!): Boolean!
     resetPassword(password: String!, resetPasswordToken: String!): AuthPayload!
     deleteUser(userId: String!): User!
+    createFeed(title: String!): Feed!
+    createSubscriptionPlan(type: String!): SubscriptionPlan!
```

Comment:
truthfully in real life creating a subscription plan would probably be a lot more complicated. in this function you would need to accept payment information too. it's a good start though.

the endpoint should probably be renamed `signupUser` too, and it would take a `type` as a param, and also any information needed to process the payment.